apps/api/middleware/checkToken.ts

```diff
@@ -0,0 +1,30 @@
+import admin from 'firebase-admin';
+
+import serviceAccount from '../service-account-key.json';
+
+admin.initializeApp({
+	credential: admin.credential.cert({
+		projectId: serviceAccount.project_id,
+		clientEmail: serviceAccount.client_email,
+		privateKey: serviceAccount.private_key,
+	}),
+});
+
+export const checkToken = async (req, res, next) => {
```

Comment:
naming proposal, just 2 cent thoughts, feel free to ignore this:
- if we use `checkToken`, it should explain what kind of thing we're checking on the token - e.g `checkTokenValid`
- or use `verifyToken` which self-explains the action better.